src/app/asign-family/asign-family.component.ts

```diff
@@ -77,7 +77,7 @@ export class AsignFamilyComponent implements OnInit, OnDestroy {
 
         if (this.isValidPhone()) {
             this.phone = cleanPhone;
-            let helper = await this.context.for(Helpers).findFirst(h => h.phone.isEqualTo(this.phone));
+            let helper = await this.context.for(Helpers).findFirst(h => h.phone.isEqualTo(this.phone).and(h.active()));
```

Comment:
Why? If the helper is inactive - but is coming to volunteer - we should be able to reactive them.
The way it is now will cause a duplicate phone number on login

Reply:
I am trying to remove them form the search but could not find how. 
currently was only able to remove in helpers-component so they do not appear in the list. 
the logic for the search is too complex for my feeble mind :) - can you show me?